ping.py

```diff
@@ -63,19 +65,121 @@ def get_ident(): return 0
 MAX_SLEEP = 1000
 
 
-class MStats:
-    thisIP = "0.0.0.0"
-    pktsSent = 0
-    pktsRcvd = 0
-    minTime = 999999999
-    maxTime = 0
-    totTime = 0
-    avrgTime = 0
-    fracLoss = 1.0
+# noinspection PyAttributeOutsideInit
```

Comment:
You know you can ignore inspection warnings for specific projects in PyCharm right?

Thats what I tend to do if the repo I'm working on blows up PyCharm.